Sprint-2/debug/1.js

```diff
@@ -6,3 +6,17 @@ function sum(a, b) {
 }
 
 console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
+
+Prediction: theres something wrong with the syntax of our code, infront of our return we have a semi Colon instead of a colon and our alignment is off. 
```

Comment:
after running the code on node I saw that the reason maybe it was not running was that we broke it off early, upon reading about it I saw the semi-colon ends our function. so the reason was it could not get to a and b, which disrupts the whole calculation.

Reply:
I don't think the semi colon ends the function. What are the rules for return statements? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return

Reply:
Oh okay I now I understand, in the link you sent, it is said that one of the rules of the return statement is its forbids line terminators between the return keyword and the expression to be returned. Thank you for the article I am reading more about these rules and how they apply